api/proto/disperser/v2/disperser_v2.proto

```diff
@@ -178,14 +190,23 @@ message GetPaymentStateForAllQuorumsReply {
   // payment vault parameters with per-quorum configurations
   PaymentVaultParams payment_vault_params = 1;
   // period_records maps quorum IDs to the off-chain account reservation usage records for the current and next two periods
+  // Should contain the same number of entries as the `reservations` field.
   map<uint32, PeriodRecords> period_records = 2;
   // reservations maps quorum IDs to the on-chain account reservation record
+  // Should contain the same number of entries as the `period_records` field.
   map<uint32, QuorumReservation> reservations = 3;
-  // off-chain on-demand payment usage. This field is currently only tracked by EigenLabs disperser because on-demand requests are only
-  // supported by EigenLabs. Future work will support decentralized on-demand dispersals and this field later be tracked and shared by
-  // dispersers unlimited to EigenLabs.
+  // off-chain on-demand payment usage.
+  // The bytes are parsed to a big.Int value.
+  // This value should always be <= cumulative_payment, as the disperser cumulative_payment kept offchain is only periodically updated onchain.
```

Comment:
this description isn't accurate atm. updating the onchain balance is something we have been putting off for the future. There's some proposal on how it updates, verifies, and handling refunds... but nothing has been decided

Reply:
[f10d69a](https://github.com/Layr-Labs/eigenda/pull/1686/commits/f10d69a806d7c37109a87ca79aa5e1b28cfcf18b)